types/case-sensitive-paths-webpack-plugin/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for case-sensitive-paths-webpack-plugin 2.1
+// Project: https://github.com/Urthen/case-sensitive-paths-webpack-plugin#readme
+// Definitions by: Andrew Makarov <https://github.com/r3nya>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { Plugin } from 'webpack';
+
+export = CaseSensitivePathsWebpackPlugin;
+
+declare class CaseSensitivePathsWebpackPlugin extends Plugin {
+    constructor(options?: CaseSensitivePathsWebpackPlugin.Options);
+}
+
+declare namespace CaseSensitivePathsWebpackPlugin {
+    interface Options {
```

Comment:
I think it will still work correctly to have `Options` at the top-level. Can you add a test for it and see whether it works?

Reply:
That is, I don't think `namespace CaseSensitivePathsWebpackPlugin` is needed.

Reply:
@sandersn ah, ok. Will do it soon. Thanks!

Reply:
Actually, after thinking a bit, it probably does need to be inside the namespace. Either way, a test would be nice.

Reply:
@sandersn sorry, but how I can test it? You mean that I need to move `Options` into the top-level?

Reply:
Add a usage of `Options` in the tests file, something like `let options: Options = { debug: true }`. I think you'll need to refer to it as `CaseSensitivePathsWebpackPlugin.Options`, though.